### ISSUE TITLE
DATA DOWNLOAD

### ISSUE THREAD

**Issue by turkfuat** [junqiangchen/KiTS19-Challege#3]
How can ı download

**Comment by MinuteswithMetrics** [comment]
You can get the data from here: 

https://github.com/neheller/kits19

**Comment by turkfuat** [comment]
What are the features of the system you are running?
(minimum requirements of the system)
iPhone’umdan gönderildi

MinuteswithMetrics <notifications@github.com<mailto:notifications@github.com>> şunları yazdı (2 May 2019 21:10):


You can get the data from here:

https://github.com/neheller/kits19

**Comment by MinuteswithMetrics** [comment]
I believe the author of the code is using python 2.7 and tf 1.8 w/GPU. I'm using python 3.6 and tf v. 1.11. His instructions are in the readme.md.

**Comment by turkfuat** [comment]
actually I mean the property of your information,I think I'm misunderstood

________________________________
Gönderen: MinuteswithMetrics <notifications@github.com>
Gönderildi: 3 Mayıs 2019 Cuma 17:54
Kime: junqiangchen/KiTS19-Challege
Bilgi: turkfuat; Author
Konu: Re: [junqiangchen/KiTS19-Challege] DATA DOWNLOAD (#3)


I believe the author of the code is using python 2.7 and tf 1.8 w/GPU. I'm using python 3.6 and tf v. 1.11. His instructions are in the readme.md.

**Comment by turkfuat** [comment]
Your computer CPU-GPU-RAM ?

iPhone’umdan gönderildi

fuat türk <turk_fuat@hotmail.com<mailto:turk_fuat@hotmail.com>> şunları yazdı (3 May 2019 23:13):

actually I mean the property of your information,I think I'm misunderstood

________________________________
Gönderen: MinuteswithMetrics <notifications@github.com<mailto:notifications@github.com>>
Gönderildi: 3 Mayıs 2019 Cuma 17:54
Kime: junqiangchen/KiTS19-Challege
Bilgi: turkfuat; Author
Konu: Re: [junqiangchen/KiTS19-Challege] DATA DOWNLOAD (#3)


I believe the author of the code is using python 2.7 and tf 1.8 w/GPU. I'm using python 3.6 and tf v. 1.11. His instructions are in the readme.md.

**Comment by MinuteswithMetrics** [comment]
GPU. My plan is to use colab since it's free.

**Comment by turkfuat** [comment]
hello ;

how to generate train and test csv ?

run utils.py  but I can't generate trainsegmantation.csv...

**Comment by turkfuat** [comment]
how to run utils.py ?      and        how to run subset.py?  Can you give sample.

**Comment by MinuteswithMetrics** [comment]
This is how I ran subset:

`!python subset.py -s 1 traintumorSegmentation.csv 200 train_2d.csv test_2d.csv`

I ran utils.py in notebook to get both `traintumorSegmentation.csv `and `trainSegmentation.csv`

**Comment by turkfuat** [comment]
Thank you very much for your work. really useful, but I'll have a few questions;

1. How many steps in vnet3d train.py will be enough for training,ex. 2000,3000,....

2.  why  train accuracy and train loss analysis  is bad

3. How to draw train accuracy and train loss graphs.

thanks.

**Comment by junqiangchen** [comment]
1、training step can set 10 or 20 epoch
2、maybe batchsize is too little,so train accuracy and train loss are unstable
3、use tensorboard can show the train accuracy and train loss graphs
